dev/io.openliberty.microprofile.openapi.2.0.internal_fat_tck/publish/tckRunner/pom.xml

```diff
@@ -1,71 +1,118 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-    Copyright (c) 2018 IBM Corporation and others. All rights reserved.
-    This program and the accompanying materials are made available under the 
-    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
-    and is available at 
-        http://www.eclipse.org/legal/epl-v10.html 
-    Contributors: 
-        IBM Corporation - initial API and implementation
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
     <groupId>io.openliberty.microprofile.openapi20.internal</groupId>
-    <artifactId>tck.runner</artifactId>
+    <artifactId>tck.runner.tck</artifactId>
     <version>2.0-SNAPSHOT</version>
-    <packaging>pom</packaging>
+    <packaging>jar</packaging>
+
     <name>MicroProfile OpenAPI TCK Runner</name>
+    <url>https://github.ibm.com/</url>
 
     <properties>
-        <arquillian.version>1.3.0.Final</arquillian.version>
+        <microprofile.openapi.version>2.0</microprofile.openapi.version>
+
+        <surefire.version>2.17</surefire.version> <!-- Any changes to the surefire version must be tested against ZOS-->
+        <arquillian.version>1.6.0.Final</arquillian.version>
+        <arquillian-wlp.version>1.0.8</arquillian-wlp.version>
+
+        <suiteXmlFile>${defaultSuiteFiles}</suiteXmlFile>
+        <targetDirectory>${project.basedir}/target</targetDirectory>
+        <testng.version>6.14.3</testng.version>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
 
-    <modules>
-        <module>tck</module>
-    </modules>
-
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
-                <version>1.2</version>
-            </dependency>
-            <dependency>
-                <groupId>org.jboss.arquillian</groupId>
-                <artifactId>arquillian-bom</artifactId>
-                <version>${arquillian.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
+                <groupId>org.jboss.spec.javax.servlet</groupId>
+                <artifactId>jboss-servlet-api_3.0_spec</artifactId>
```

Comment:
Is this the servlet API? If so, is there a liberty servlet API that is equivalent or is it actually using jboss classes?

Reply:
It's the servlet API which is pulled in by `liberty-arquillian-managed`. IIRC, liberty-arquillian-managed specifies an acceptable version range and for some reason when the maven build runs on zOS, it has trouble fetching or reading the metadata which would allow it to find the latest available version which matches its range. We're working around that by specifying a particular version for this artifact here.